src/app/shared/components/histogram-bucket-form-group/histogram-bucket-form-group.component.scss

```diff
@@ -20,6 +20,9 @@ under the License.
 .table {
     display: flex;
     justify-content: space-between;
+    &-body-right {
```

Comment:
Why do you have to add that ? The name is not very explicit I feel like

Reply:
for one view the input is not well displayed. So i added that to fixe this behavior. I admit i do not have an explicit name for that. Because the component is made of 2 table body to display the content.